starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Apple Pie Recipe</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/main.css">
 </head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+<body>
+  <div id="jumbatron">
+    <h1>Apple Pie</h1>
+  </div>
 
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
+  <main>
+    <section class="note">
+      </p>This was my grandmother's apple pie recipe. I have never seen another one
+      quite like it. It will always be my favorite and has won me several first place
+      prizes in local competitions. I hope it becomes one of your favorites as well!
+      <p>
+      <img src="./images/cooking-info.png" alt="cooking-info" class = "small-img" srcset="">
+      <hr>
+    </section>
+  
+    <section class="ingredients">
+      <h2>Ingredients</h2>
+      <ul>
+        <li> 1 recipe pastry for a 9 inch double crust pie</li>
+        <li> 1/2 cup unsalted butter</li>
+        <li> 3 tablespoons all-purpose flour</li>
+        <li> 1/4 cup water</li>
+        <li> 3 tablespoons all-purpose flour</li>
+        <li> 1/4 cup water</li>
+        <li> 1/2 cup white sugar</li>
+        <li> 1/2 cup packed brown sugar</li>
+        <li> 8 Granny Smith apples - peeled, cored and sliced</li>
+      </ul>
+      <img src="./images/recipe-info.png" alt="" class = "small-img" srcset="">
+      <hr>
```

Comment:
If you place you < hr > elements outside of sections, they will be both the same height (on the contrary, the sections styles apply to them, and they might be different)